include/zotonic_notifications.hrl

```diff
@@ -642,6 +642,11 @@
         state :: term()
     }).
 
+%% @doc Notify modules of a data model entry defined in manage_schema
+-record(managed_data, {
+    module :: atom(),
+    props :: list()
+}).
 
```

Comment:
Changed to `manage_data`. 

> Also, maybe also include the whole #datamodel{} here (and datamodel version)

You mean the `mod_schema` version? We don’t have that in the place where we send the notification (`z_datamodel`).